src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/BehandlingService.kt

```diff
@@ -43,6 +43,16 @@ class BehandlingService(
         .filter { !it.erHenlagt() && it.status == BehandlingStatus.AVSLUTTET }
         .maxByOrNull { it.opprettetTidspunkt }
 
+    /**
+     * Henter siste iverksatte behandling FØR en gitt behandling.
+     * Bør kun brukes i forbindelse med oppdrag mot økonomisystemet
+     * eller ved behandlingsresultat.
+     */
+    fun hentForrigeBehandlingSomErIverksatt(behandling: Behandling): Behandling? {
```

Comment:
Siden denne metoden brukes kun for å generere utbetalingsoppdrag, er det ikke riktig å hente alltid siste iverksatt behandling?

Reply:
Snakket med Eivind om dette nå og det høres ut som at vi alltid kan bruke siste iverksatte/vedtatte behandling som du sier 👍 Fjerner derfor metoden ☝️  og bruker heller `hentSisteBehandlingSomErVedtatt` som du har lagt inn!